connect/mirror/src/main/java/org/apache/kafka/connect/mirror/Scheduler.java

```diff
@@ -70,6 +70,10 @@ void execute(Task task, String description) {
         }
     } 
 
+    void executeAsync(Task task, String description) {
+        executor.submit(() -> executeThread(task, description));
```

Comment:
Looking at the body of `executeThread`, won't this still cause a warning log message to be emitted if the task takes too long to execute?

```java
    private void executeThread(Task task, String description) {
        // ...
        run(task, description);
    }

    private void run(Task task, String description) {
        try {
            long start = System.currentTimeMillis();
            task.run();
            long elapsed = System.currentTimeMillis() - start;
            LOG.info("{} took {} ms", description, elapsed);
            if (elapsed > timeout.toMillis()) {
                // Warning message gets logged here
                LOG.warn("{} took too long ({} ms) running task: {}", name, elapsed, description);
            }
        }
        // ...
    }
```

Reply:
Removed logging for the `executeAsync` method.
Let me know what you think.